sdk-workflows/pom.xml

```diff
@@ -1,6 +1,6 @@
 <project
-    xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
@artur-ciocanu there is no change in here.. can we make sure we don't change this file?

Reply:
Reverted